source/MaterialXRender/OpenGL/GLTextureHandler.cpp

```diff
@@ -8,6 +8,11 @@ namespace MaterialX
 bool GLTextureHandler::createColorImage(float color[4],
                                         ImageDesc& imageDesc)
 {
+    if (!glActiveTexture)
```

Comment:
Small change for MaterialXView which can use this as a stand-alone without initialization via validation. If the function pointer hasn't been initialized then we need to initialize it.